tests/unit/cli/vyper_json/test_output_selection.py

```diff
@@ -41,7 +52,23 @@ def test_star():
         "sources": {"foo.vy": "", "bar.vy": ""},
         "settings": {"outputSelection": {"*": ["*"]}},
     }
-    expected = sorted(set(TRANSLATE_MAP.values()))
+    translate_map = set(TRANSLATE_MAP.values())
+    # if the venom flag is not present
+    translate_map.remove("bb")
+    translate_map.remove("bb_runtime")
+    translate_map.remove("cfg")
+    translate_map.remove("cfg_runtime")
```

Comment:
style, bit more compact:
```suggestion
    venom_outputs = ("bb", "bb_runtime", "cfg", "cfg_runtime")
    for s in venom_outputs:
        translate_map.remove(s)
```